elfloader-tool/src/arch-arm/64/mmu.c

```diff
@@ -11,6 +11,8 @@
 #include <mode/structures.h>
 #include <printf.h>
 #include <abort.h>
+#include <mode/aarch64.h>
+#include <armv/machine.h>
```

Comment:
Why do we include this file now also, was it missing before?

Reply:
I believe it's for some of the constants such as `MT_DEVICE_nGnRnE` or this change is just an artifact of splitting up the previous PR.